sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -461,68 +462,246 @@ case class FindInSet(left: Expression, right: Expression) extends BinaryExpressi
 }
 
 /**
- * A function that trim the spaces from both ends for the specified string.
+ * A function that trims leading or trailing characters (or both) from the specified string.
```

Comment:
Could you add a comment for the parameter? And also for `StringTrimLeft` and `StringTrimRight`.